opencti-platform/opencti-front/src/private/components/data/ingestionCsv/IngestionCsvCreation.tsx

```diff
@@ -286,11 +286,9 @@ const IngestionCsvCreation: FunctionComponent<IngestionCsvCreationProps> = ({ pa
               )}
               <CreatorField
                 name="user_id"
-                label={t_i18n(
-                  'User responsible for data creation (empty = System)',
-                )}
-                isOptionEqualToValue={(option: Option, value: string) => option.value === value}
+                label={t_i18n('User responsible for data creation (empty = System)')}
                 containerStyle={fieldSpacingContainerStyle}
+                showConfidence
```

Comment:
I went for an opt-in optional display, even if in most cases we want it displayed.

I'm generally in favor of explicit opt-in options for clarity ; but happy to discuss if you think otherwise.